packages/client/src/types.js

```diff
@@ -295,6 +301,22 @@ export type IdentityCreateUserParams = {
 
 export type IdentityCreateResult = { id: ID }
 
+export type IdentityCreateContactParams = {
+  userId: string,
```

Comment:
Could we set this type to `userID` please for consistency with the other occurrences?